src/frontend/planner_test/tests/testdata/bushy_join.yaml

```diff
@@ -359,32 +359,32 @@
     └─StreamProject { exprs: [nation.n_name, sum($expr1)] }
       └─StreamHashAgg { group_key: [nation.n_name], aggs: [sum($expr1), count] }
         └─StreamExchange { dist: HashShard(nation.n_name) }
-          └─StreamProject { exprs: [nation.n_name, (lineitem.l_extendedprice * (1:Int32 - lineitem.l_discount)) as $expr1, lineitem.l_orderkey, lineitem.l_linenumber, customer.c_custkey, orders.o_orderkey, region.r_regionkey, nation.n_nationkey, supplier.s_suppkey, lineitem.l_suppkey, customer.c_nationkey] }
```

Comment:
We copy the whole `tpch.yaml` file to verify the bushy join reordering, but some of them are useless, could you please remove those cases? Additionally, could you provide a test case comprising of no less than 20 inner joins to validate if this present algorithm can efficiently handle large-scale join operations?